### ISSUE TITLE
Which kubo docker image can be used for plugin release "go-ds-s3-plugin/v0.32.1"?

### ISSUE THREAD

**Issue by Marco-DKRZ** [ipfs/go-ds-s3#294]
We are using the docker image `ipfs/kubo:v0.32.1`  and received the following error message when using the plugin `go-ds-s3-plugin/v0.32.1`:
`Error: error loading plugins: loading plugin /data/ipfs/plugins/go-ds-s3-plugin: plugin.Open("/data/ipfs/plugins/go-ds-s3-plugin"): plugin was built with a different version of package github.com/ipfs/kubo/core/coreiface/options
exit code: 1`

If we want to use an existing docker image, which one can we use?

**Comment by hsanjuan** [comment]
it could be that the docker-image was built with a slightly different go version than the dist.ipfs.io builds

**Comment by Marco-DKRZ** [comment]
> On linux/amd64?

Yes, we used  go-ds-s3-plugin_v0.32.1_linux_amd64.tar.gz

**Comment by hsanjuan** [comment]
So, I can confirm this happens.

I can also confirm that it works with the dist version of v0.32.1. And I have no idea why. I don't manage to build myself a Kubo version + plugin that works, or a Docker build that works. I have no idea tbh.

**Comment by Marco-DKRZ** [comment]
Yes, I can confirm that we managed to get it working with the dist version (https://dist.ipfs.tech/kubo/v0.32.1/kubo_v0.32.1_linux-amd64.tar.gz).
Thank you for your hint!

**Comment by gammazero** [comment]
Building plugins must be done carefully. Here is some information on that: https://github.com/ipfs/go-ipfs-example-plugin?tab=readme-ov-file#building-and-installing

If the issue reported here is solved, please close this issue.
